### ISSUE TITLE
what is the real and fake?

### ISSUE THREAD

**Issue by Wentao795** [znxlwm/pytorch-CartoonGAN#9]
D_real_loss = BCE_loss(D_real, real)
D_fake_loss = BCE_loss(D_fake, fake)
why use loss to optim them to real and fake?
